test_end_to_end/test_page_titles.py

```diff
@@ -0,0 +1,26 @@
+# see: https://playwright.dev/python/docs/test-runners
+# pytest --browser webkit --headed
+
+import re
+
+import pytest
+from playwright.sync_api import Page, expect
+
+
+def test_homepage_has_Playwright_in_title_and_get_started_link_linking_to_the_intro_page(page: Page):
```

Comment:
maybe make this name shorter but as it's the same as what you showed during the session let's leave it like this for now